.github/workflows/release.yml

```diff
@@ -0,0 +1,119 @@
+          # Install twine
+          python3 -m pip install setuptools wheel twine
+
+      # Upload to TestPyPI
+      - name: Build and Upload to TestPyPI
+        if: ${{ inputs.upload_server == 'testpypi'}}
+        run: |
+          python3 setup.py sdist --formats=zip
+          twine check dist/* --strict 
+          python3 -m twine upload dist/*
+        env:
+          TWINE_USERNAME: __token__
+          TWINE_PASSWORD: ${{ secrets.TESTPYPI }}
+          TWINE_REPOSITORY: testpypi
+
+      # Upload to PyPI
+      - name: Build and Upload to PyPI
+        if: ${{ inputs.upload_server == 'pypi' }}
+        run: |
+          python3 setup.py sdist --formats=zip
+          twine check dist/* --strict 
+          python3 -m twine upload dist/*
+        env:
+          TWINE_USERNAME: __token__
+          TWINE_PASSWORD: ${{ secrets.PYPI }}
+          TWINE_REPOSITORY: pypi
+
+      - name: Sleep for 300s to make release available
+        uses: juliangruber/sleep-action@v1
+        with:
+          time: 300s
+
+
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+        os:  [ ubuntu-latest, windows-latest ]
+        group: [ 1, 2 ]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install pytest nbmake pytest-xdist python-igraph pytest-split numba seaborn matplotlib plotly geopandas ortools xlsxwriter openpyxl cryptography psycopg2 matpowercaseframes
+          if ${{ matrix.python-version == '3.7' }}; then python -m pip install pypower; fi
+          if ${{ matrix.python-version != '3.7' }}; then python -m pip install numba; fi
+          if ${{ matrix.python-version == '3.7' }}; then python -m pip install lightsim2grid; fi
+          if ${{ matrix.python-version == '3.10' }}; then python -m pip install lightsim2grid; fi
+      - name: Install Julia
+        if: ${{ matrix.python-version == '3.7' }}
+        run: |
+          ./.install_julia.sh 1.5
+          pip install julia
+          python ./.install_pycall.py
+      - name: Install pandapower from TestPyPI
+        if: ${{ inputs.upload_server == 'testpypi'}}
+        run: |
+          pip install --no-cache-dir -i https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple pandapower
+      - name: Install pandapower from PyPI
+        if: ${{ inputs.upload_server == 'pypi'}}
+        run: |
+          pip install pandapower
+      - name: List all installed packages
+        run: |
+          pip list
+      - name: Test with pytest
+        run: |
+          pytest --splits 2 --group ${{ matrix.group }}
+      - name: Test tutorials
+        run: |
+          pytest --nbmake -n=auto "./tutorials"
```

Comment:
I'm wondering where these tutorials come from, since they are not included in the pypi package. Maybe from line 30?

Reply:
I agree on that. Adapted it and removed the tutorials test